pulser-pasqal/pulser_pasqal/backends.py

```diff
@@ -122,22 +128,14 @@ class EmuTNBackend(PasqalEmulator):
     Args:
         sequence: The sequence to send to the backend.
         connection: An open PasqalCloud connection.
-        config: An EmulatorConfig to configure the backend.
+        config: An EmulatorConfig to configure the backend. If not provided,
+            the default configuration is used.
     """
 
     emulator = pasqal_cloud.EmulatorType.EMU_TN
     default_config = DEFAULT_CONFIG_EMU_TN
     configurable_fields = ("backend_options", "sampling_rate")
```

Comment:
What is the configuration `backend_options` here?

Reply:
Don't you want to define the options that can be defined in these `backend_options` for each `PasqalEmulator` (and restrict the keys of `backend_options` to these options) ? 
Edit: I guess that's being done in `pasqal_cloud.EmulatorType.EMU_TN`, but where can I find this ? 😅

Reply:
I wrote them in the docstring of each of the backends, do you think it's not enough?

Reply:
Okay I have found out where the test is made ! (in`_convert_configuration`) no it's okay it's clear.